GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_all_columns.yaml

```diff
@@ -0,0 +1,8 @@
+subclass_name: dc2_object.DC2ObjectCatalog
+base_dir: /global/projecta/projectdirs/lsst/global/in2p3/Run2.1i/w_2019_19-v1/dpdd/calexp-v1:coadd-dr1b-v1/object_table_summary
+schema_filename: trim_schema.yaml
+filename_pattern: 'object_tract_\d+\.hdf5$'
+description: DC2 Run 2.1i Object Catalog (with only DPDD columns and native columns needed for the DPDD columns)
+creators: ['DESC DC2 Team']
+pixel_scale: 0.2
+include_in_default_catalog_list: true
```

Comment:
Do we want to include this catalog in the default list?

Reply:
No.  Fixed.